### PR TITLE
Debian9 ubuntu16

### PR DESCRIPTION
**Pull request by Skywalker-11**
Includes PR #20:
- split configuration to multiple files
- configuration file for install
- compatibility with debian 9, ubuntu 16.x and 17.x

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -47,9 +47,9 @@ install-mailscanner
 
 ##ask directory for web files
 logprint ""
-ask "In what location should MailWatch be installed (web files directory)?[/var/www/html/mailscanner/]:" WebFolder
+ask "In what location should MailWatch be installed (web files directory)?[/opt/mailwatch/]:" WebFolder
 if [ -z $WebFolder ]; then
-    WebFolder="/var/www/html/mailscanner/"
+    WebFolder="/opt/mailwatch/"
```
Comment (stefaweb) [inline setup.sh]: May be `/opt/mailwatch/web` will be better?

If later we add other files or directory in /opt/mailwatch (as /opt/mailwatch/bin for all mailwatch_xxx.php files).

Comment (Skywalker-11) [reply setup.sh]: Your right. 
@endelwar did you already decided on the new locations under /opt/mailwatch?

Comment (endelwar) [reply setup.sh]: it will be `/opt/mailwatch/public` following php-pds recommendation (https://github.com/php-pds/skeleton)

Comment (Skywalker-11) [reply setup.sh]: done

**Pull request merged**